examples/python/0_basic/cuda/basic_example_gpt_oss.py

```
@@ -0,0 +1,248 @@
+
+
+def check_transformers_version():
+    import transformers
+    # GPT-OSS config requires a recent transformers version
+    required_version = "4.50.0"
+    current_version = transformers.__version__
+    if current_version < required_version:
+        raise RuntimeError(
+            f"Transformers {current_version} is too old; GPT-OSS requires >={required_version}. "
+            "Run: pip install 'transformers>=4.50.0'"
+        )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)